### PR TITLE
init

### PR DESCRIPTION
**Pull request by Ninroot**
Initialize repository with Dailymotion standards. The PR also includes the fix of the [missing browser version](https://github.com/dailymotion-oss/udger/pull/1/files#diff-070ce9f27c07570a03f9e1f9c20cd188e552f697e367f2a6e8cd7bfa184cf92cL141).

### CODE REVIEW

File: udger.go [hunk 1]
```diff
@@ -8,29 +8,14 @@ import (
...
-func New(dbPath string, flags *Flags) (*Udger, error) {
-	if flags == nil {
-		flags = &Flags{
-			browser: true,
-			Device:  true,
-			os:      true,
-		}
-	}
+// New creates a new instance of Udger from the dbPath database loaded in memory for fast lookup.
+func New(dbPath string) (*Udger, error) {
```
Comment (sebmil-daily) [inline udger.go:12]: :+1:

File: udger.go [hunk 2]
```diff
@@ -52,66 +37,51 @@ func New(dbPath string, flags *Flags) (*Udger, error) {
...
-			return nil, err
-		}
-		info.Browser = udger.Browsers[browserID]
-		if info.Browser.Family != "" {
-			info.Browser.Name = info.Browser.Family + " " + version
-		}
-		info.Browser.Version = version
-		info.Browser.Type = udger.browserTypes[info.Browser.typ]
+	browserID, browserVersion := udger.findDataWithVersion(ua, udger.rexBrowsers, true)
+	info.Browser = udger.Browsers[browserID]
```
Comment (Aditi0101) [inline udger.go]: We are not handling the case when browserID = -1.

File: udger_test.go [hunk 3]
```diff
@@ -125,3 +138,17 @@ func TestValidDbName(t *testing.T) {
 		})
 	})
 }
+
+func BenchmarkUdger_Lookup(b *testing.B) {
```
Comment (sebmil-daily) [inline udger_test.go:166]: :+1:

**Review by sebmil-daily — APPROVED** [review]

File: udger.go (hunk 2 above)
Comment (Ninroot) [reply udger.go]: Empty browser is used in that case. But I added a unit test to check the lookup with an empty string and found out the returned value was not empty. I fixed it.

**Review by Aditi0101 — APPROVED** [review]

**Review by Aditi0101 — APPROVED** [review]

**Review by Aditi0101 — APPROVED** [review]
approved

**Review by florianorpeliere — APPROVED** [review]

**Pull request merged**